mithril.js

```diff
@@ -1125,6 +1124,6 @@ m.redraw = redrawService.publish
 m.request = requestService.xhr
 m.jsonp = requestService.jsonp
 m.version = "1.0.0"
-module.exports = m
+window.m = m
```

Comment:
Node, Borwserify and WebPack can `require('mithril')` which points to `index.js`, not `mithril.js`.